.github/workflows/test_simple.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# fail on non-zero return code from a subprocess
+set -e
+
+# print commands
+set -x
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 GDAL_AUTOLOAD_DIR"
+    exit 1
+fi
+
+# non-existent variables as an errors
+set -u
+
+export GDAL_AUTOLOAD_DIR=$1
+export GDAL_DRIVER_PATH=${GDAL_AUTOLOAD_DIR}
+
+# add small GRASS GIS dataset for tests
+(mkdir -p $HOME/grassdata && \
+ cd $HOME/grassdata/ && \
+ wget --quiet https://grass.osgeo.org/sampledata/north_carolina/nc_spm_08_micro.zip && \
+ unzip nc_spm_08_micro.zip && \
+ rm -f nc_spm_08_micro.zip )
+
+# test GRASS GIS raster map
+gdalinfo $HOME/grassdata/nc_spm_08_micro/PERMANENT/cellhd/elevation
```

Comment:
Set `LD_LIBRARY_PATH` here to make the test pass with current and older GRASS versions?

Reply:
Will try!